### DISCUSSION TITLE
Add Crazy's Dave Store

### DISCUSSION THREAD

**Discussion by TheMysteriousPapu** [ROBlNET13/pvz#64] [General]

**Comment by ROBlNET13** [comment]
Zen garden might be a little too hard to add but it could be possible. Crazy daves also might be possible and i have been thinking about it. I'll keep this suggestion in mind

**Comment by clay-rip** [comment]
Reopening to track progress

**Comment by hamhimdev** [comment]
vroom vroom

**Comment by TheUnium** [comment]
vroooommmmmm

**Comment by hamhimdev** [comment]
Vroom Vroom

**Comment by ROBlNET13** [comment]
nyyooooom

**Comment by hamhimdev** [comment]
Vroomathon

**Comment by TheUnium** [comment]
VVVVVVVVVVRRRRRRRRRRRRRRRroooommmmmm

**Comment by clay-rip** [comment]
nroooooom

**Comment by hamhimdev** [comment]
VROOM

**Comment by TheUnium** [comment]
VROOOMMMMMM DRA DRA DRA VROOMMMMMMMMMMMMMMMMMMMMM

**Comment by clay-rip** [comment]
VYOOOM!!!

**Comment by hamhimdev** [comment]
vroom.... Vrooom... tssk tskk skkk Vroooooom!

**Comment by ROBlNET13** [comment]
They aren't just car sounds -- they're a display of years of engineering.

**Comment by clay-rip** [comment]
brbrbrbrbrbr i think its broken

**Comment by TheUnium** [comment]
vroommmmmm BOOM BOOM BOOM vroooooooooooooommmmmmmmmmmmmmmmmmm

**Comment by hamhimdev** [comment]
i hate you clay why did you make this a discussion you are so evil

**Comment by ROBlNET13** [comment]
n
y
O
oMM
m

**Comment by hamhimdev** [comment]
when codeberg repo

**Reply by clay-rip** [reply]
https://codeberg.org/pvzm/game

**Comment by ROBlNET13** [comment]
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom
nyoom

**Comment by hamhimdev** [comment]
oh yeah idk if i ever mentioned but i have it mirrored as well https://git.loxodrome.app/hamhim/game

**Comment by hamhimdev** [comment]
<img width="550" height="140" alt="image" src="https://github.com/user-attachments/assets/ff67705c-73b7-4e85-9f12-bb20f0a7e17a" />

colon worried colon
